src/pages/Account/AccountSecurity.tsx

```diff
@@ -1,11 +1,71 @@
 import * as React from 'react';
+import { ListGroup, ListGroupItem, Alert } from 'reactstrap';
+import { resetPassword } from 'utils/firebase';
+import { getDisplaySuccessMessage, SuccessMessage, getDisplayErrorMessage, ErrorMessage } from 'utils/validators';
+import { FirebaseConsumer, FirebaseUserProps } from 'HOCs/FirebaseProvider';
+import { User } from 'firebase';
 
-class AccountSecurity extends React.Component<any> {
-  render() {
-    return (
-      <h1>This is the Security Page</h1>
-    );
+interface AlertProperties {
+  color: string;
+  msg: string;
+  show: boolean;
+}
+
+function AccountSecurity() {
+  const [alert, setAlert] = React.useState<AlertProperties>({ color: '', msg: '', show: false });
```

Comment:
initially had `msg` dictate whether to show or not, but tried to make this component fade away to mimic the snackbar by using the `<Alert transition={...}>` prop, and thought `show` might help achieve fading. This current implementation has the alert with no fade

Reply:
update: using `isOpen` prop instead of conditionally rendering the component. Fade should work